logging/efk-logging-driver.adoc

```diff
@@ -1,14 +1,15 @@
 :context: efk-logging-driver
 [id='efk-logging-driver_{context}']
-= Setting the EFK logging driver
+= Setting the cluster logging driver
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
-By default, the *json-file* log driver is used by default for Docker in {product-title} as of {product-title} 3.9.
-However, if you have upgraded from {product-title} 3.7, your cluster might be configured to use the *journald* log driver configuration. 
+By default, the *json-file* log driver is used by default for Docker in 
```

Comment:
This will need to be updated again eventually. You can't upgrade, and there's no Docker.